tests/getAuthors.test.js

```diff
@@ -22,7 +22,7 @@ const Authors = {
   },
   data: [
     { id: 1, username: 'Burindi', email: 'alain1@gmail.com' },
-    { id: 43, username: 'dianeMurekatete', email: 'diane@gmail.com' }
+    { id: 44, username: 'dianeMurekatete', email: 'diane@gmail.com' }
```

Comment:
@truestbyheart why do you guys keep changing these variables used in the tests?

Reply:
This is because some use seeds as a way of creating dummy data to test and others prefer creating this data by simulating a create request then testing, this leads to some test to fail when others rebase and tweaking the seeds is the next best choice.